src/Plugin/Condition/DrupalUserExist.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\civicrm_entity\Plugin\Condition;
+
+use Drupal\rules\Core\RulesConditionBase;
+use Drupal\user\UserInterface;
+use Drupal\civicrm\Civicrm;
+
+
+/**
+ * Provides a 'Drupal linked User exist' condition.
+ *
+ * @Condition(
+ *   id = "rules_drupal_user_exist",
```

Comment:
Yeah, somewhat minor as but is it okay to prefix it with `civicrm_entity_` instead - similar to [this](https://github.com/eileenmcnaughton/civicrm_entity/blob/8.x-3.x/src/Plugin/RulesAction/UserCreate.php#L17)?